File: distancia de puntos/distancia de puntos/Form1.vb
Public Class Form1
    Private Sub TextBox1_TextChanged(sender As Object, e As EventArgs) Handles txtX1.TextChanged

    End Sub

    Private Sub btnLimpair_Click(sender As Object, e As EventArgs) Handles btnLimpair.Click
        txtX1.Text = ""
        txtX2.Text = ""
        txtY1.Text = ""
        txtY2.Text = ""
        txtDistancia.Text = ""

    End Sub

    Private Sub btnCalcular_Click(sender As Object, e As EventArgs) Handles btnCalcular.Click
        Dim x1 As Double
        Dim x2 As Double
        Dim y1 As Double
        Dim y2 As Double

        x1 = txtX1.Text
        x2 = txtX2.Text
        y1 = txtY1.Text
        y2 = txtY2.Text

        txtDistancia.Text = Math.Round(Math.Sqrt(Math.Pow(x2 - x1, 2) + Math.Pow(y2 - y1, 2)), 4)

    End Sub

    Private Sub btnexit_Click(sender As Object, e As EventArgs) Handles btnexit.Click
        End
    End Sub
End Class

File: DistanciaPuntosC/DistanciaPuntosC/Form1.cs
using System;
using System.Collections.Generic;
using System.ComponentModel;
using System.Data;
using System.Drawing;
using System.Linq;
using System.Text;
using System.Threading.Tasks;
using System.Windows.Forms;

namespace DistanciaPuntosC
{
    public partial class Form1 : Form
    {
        public Form1()
        {
            InitializeComponent();
        }

        private void BtnCal_Click(object sender, EventArgs e)
        {
            double d = 0;
            d = Math.Sqrt(Math.Pow(double.Parse(txtX1.Text) - double.Parse(txtX2.Text), 2) + (Math.Pow(double.Parse(txtY1 .Text) - double.Parse(txtY2 .Text), 2)));
            dis.Text = d.ToString();
        }

        private void btnsalir_Click(object sender, EventArgs e)
        {
            OnResizeEnd
        }
    }
}
